src/views/admin/PersonDetailsView.vue

```diff
@@ -163,6 +170,35 @@
             <v-col v-else-if="personStore.loading"> <v-progress-circular indeterminate></v-progress-circular></v-col>
           </v-row>
         </v-container>
+        <v-divider
+          class="border-opacity-100 rounded my-6 mx-4"
+          color="#E5EAEF"
+          thickness="6"
+        ></v-divider>
+        <!-- Zuordnungen -->
+        <v-container class="person-zuordnungen">
+          <v-row class="ml-md-16">
+            <v-col>
+              <h3 class="subtitle-1">{{ $t('person.zuordnungen') }}</h3>
+            </v-col>
+          </v-row>
+          <v-row>
```

Comment:
Die Zuordnungen sind nicht bündig mit den persönlichen Informationen weiter oben. Passt das, wenn wir die gleichen cols wie oben benutzen?

Reply:
Der title ist eigentlich bündig aber die Daten nicht. Ich dachte mir ursprünglich auch, dass wir vielleicht bei den Daten auch die gleichen Cols nutzen könnten aber es sah komisch aus, weil oben  einfach anders ist mit "key: value" und den style "text-right". Ich wollte auch die Daten hier mindestens ein cols="10" geben, damit soviel wie möglich in einer Zeile angezeigt wird.

Reply:
Ich werde falls auch Christian fragen, was er davon denkt, wenn das in Ready for test ist.